ArduPlane/mode.h

```diff
@@ -686,6 +687,9 @@ class ModeQLand : public Mode
     bool _enter() override;
     bool _pre_arm_checks(size_t buflen, char *buffer) const override { return false; }
 
+    uint32_t last_target_loc_set_ms;
```

Comment:
I wonder if this should just be a quadplane var with the others in `quadplane.poscontrol`

Reply:
that code doesn't need it, so I think it should stay in qland